force-app/main/default/classes/SummitEventsShared.cls

```diff
@@ -616,6 +616,25 @@ public with sharing class SummitEventsShared {
         return pickList;
     }
 
+    // Test to see if person accounts are enabled.
+    public static Boolean personAccountsEnabled()
+    {
+        try
+        {
+            // Try to use the isPersonAccount field.
+            sObject testObject = new Account();
+            testObject.get( 'isPersonAccount' );
+            // If we got here without an exception, return true.
+            return true;
+        }
+        catch( Exception ex )
+        {
+            // An exception was generated trying to access the isPersonAccount field
+            // so person accounts aren't enabled; return false.
+            return false;
+        }
```

Comment:
Hi, I think we could also use this option in case we don't want to fire exceptions: 
`Schema.sObjectType.Account.fields.getMap().containsKey( 'isPersonAccount' );`

I created a new scratch org and I tested both using anonymous window. 

- try-catch example takes around 71.342 ms 
- Schema approach takes around 16.284 ms

Both approaches are intuitive 👍